app/src/userpages/components/StreamPage/List/streamList.pcss

```diff
@@ -11,3 +11,9 @@
   min-width: 0 !important;
   max-width: 36px !important;
 }
+
+.streamsTable {
+  :global(.table-responsive) {
+    min-height: 500px;
```

Comment:
another option could be `min-height: stretch` (i.e. `-webkit-fill-available` after autoprefixing)

Reply:
Ahh yes, good idea, I didn't think of `stretch`. 

I think the implemented way is a little rough but might be more browser friendly. I may revisit it later.